server/models.py

```diff
@@ -104,9 +104,110 @@ class Solutions(db.Model):
     local_environmental_quality = db.Column(db.Integer)
     enhances_public_safety = db.Column(db.Integer)
     builds_resilience = db.Column(db.Integer)
+    vech_tran = db.Column(db.Integer)
+    energy = db.Column(db.Integer)
+    waste = db.Column(db.Integer)
 
 # ---------- Marshmallow schema class to serialize data --------- #
 class Zip_Data_Schema(ma.SQLAlchemyAutoSchema):
     class Meta:
         model=Zip_data
-        load_instance=True
\ No newline at end of file
+        load_instance=True
+
+# ----------- Views for recommendation Filters -------------------#
+
+class Solutions_vech_tran_energy(db.Model):
```

Comment:
I used views so that I could filter on holistic and sector at the same. If you want to figure out a better way go ahead but the code I wrote works and it too me a long time to get it to work

Reply:
You did very good! If it works it works. But it can be better by following the OOP principle of DRY- Don't Repeat Yourself.  This is important in maintaining clean and readable code, and is necessary for this project. The tables look like they can be combined.  With your permission I can help you do this!

Reply:
Okay, I'll try and look at it this afternoon. I have a grad visitation day today and a midterm Wednesday, so I'll see if I have time to make the changes you want by tomorrow since that is when we are sending it out

Reply:
I see you are busy Gillian, its OK! I was suggesting changes for the final display of the code on the master repo.  We can deploy your changes without merging them to the master if they are not ready, as it does not necessarily change the functionality of the product at this stage.  However, it is important for the code displayed in the master repo to adhere to clean code standards because it is acting as a sort of resume for you, me, and the whole team.

Reply:
Okay Eric I did what you asked, let me know if you want anything else changed. Also tomorrow we need to discuss the general code about connecting to the database because I think there are some issues which does not let me select certain columns.